src/sentry/utils/committers.py

```diff
@@ -311,9 +311,11 @@ def get_serialized_event_file_committers(
         if not owner:
             return []
         commit = Commit.objects.get(id=owner.context.get("commitId"))
+
+        serialized_owners = user_service.serialize_many(filter={"user_ids": [owner.user_id]})
         author = (
-            user_service.serialize_many(filter={"user_ids": [owner.user_id]})[0]
-            if owner.user
+            serialized_owners[0]
+            if serialized_owners
```

Comment:
I think we're better off using `if owner.user_id` here so that we don't make the call to `user_service.serialize_many` unnecessarily if it's `None`

Reply:
Got it, I added the shortcircuit.  Worth pointing out, and I added comments to this effect, that *just because there is a user_id doesn't mean you will get a result back*.  This is due to the fact that any cross silo foreign key only cascades using eventual consistency.  There will always be some gap between a user is deleted and when the associated region silo identifiers get purged.

Reply:
That's ok imo. The search would fail, but it should fail anyway since the user is deleted. It'd be a little user unfriendly, but I assume it's not going to take long to cascade in most cases